src/components/Charts.jsx

```diff
@@ -2,52 +2,51 @@
 
 import React from 'react';
 
+import Immutable from 'immutable';
+
 import ChartWithHeader from './ChartWithHeader';
 
+const DIV_STYLE = {overflow: "auto", maxHeight: "100vh"};
+
 class Charts extends React.Component {
     render() {
-        var charts = [];
+        let charts = [];
```

Comment:
`const` and `let` are both block-scoped. The [MDN docs](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/const) about `const` state at the beginning that "const" does not mean "immutable", which is what you refer. This [SO question](http://stackoverflow.com/questions/31686412/does-es6-const-affect-garbage-collection) gives some clarification about the lexical scope thing.

I'd say declare everything as const if you can :)